packages/cli/src/oclif/commands/init.js

```diff
@@ -15,11 +15,11 @@ class InitCommand extends BaseCommand {
     const env = yeoman.createEnv();
     env.registerStub(ProjectGenerator, 'zapier:integration');
 
-    env.run('zapier:integration', { path, template }, () => {
-      this.log();
-      this.log(`A new integration has been created in directory "${path}".`);
-      this.log(`Read all about it in "${join(path, 'README.md')}".`);
-    });
+    await env.run('zapier:integration', { path, template });
+
+    this.log();
+    this.log(`A new integration has been created in directory "${path}".`);
+    this.log(`Read all about it in "${join(path, 'README.md')}".`);
```

Comment:
Switching from callback to promises/async. Side note: this should have been broken in v3.0.0 (see last point in [changelog](https://github.com/yeoman/environment/releases/tag/v3.0.0)) but enforcement was broken until v3.5.0 (see last point in [changelog](https://github.com/yeoman/environment/releases/tag/v3.5.0))